docs/UserGuide.md

```diff
@@ -134,14 +134,14 @@ Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] `
     * will not match person named `john` with address `Bedok Reservoir`.
 
 Examples:
-* `find n/John Lee` matches `john`, `johnny Doe`, `Aileen`.
-* `find a/Jurong east n/Ben e/gmail` matches
+* `find n/John Lee` returns `john`, `johnny Doe`, `Aileen`.
+* `find a/Jurong east n/Ben e/gmail` returns 
     * person with name: `benny tan`, address: `West Jurong`, email: `benny.tan@gmail.com`.
     * person with name: `benjamin` address: `yishun east ave 1` email: `benj@gmail.com`.
```

Comment:
Oops I'm confused now. The address specified is Jurong east but the command will return persons who matches the name keyword but not the address keyword(s?

Reply:
it will match the word east